README.md

```diff
@@ -1,4 +1,4 @@
-[![CircleCI](https://circleci.com/gh/giantswarm/cert-exporter.svg?&style=shield&circle-token=0b5cc07c7114258992b3411a963ce9515e32106a)](https://circleci.com/gh/giantswarm/cert-exporter)
+[![CircleCI](https://circleci.com/gh/giantswarm/cert-exporter.svg?&style=svg)](https://circleci.com/gh/giantswarm/cert-exporter)
```

Comment:
```suggestion
[![CircleCI](https://circleci.com/gh/giantswarm/cert-exporter.svg?&style=shield)](https://circleci.com/gh/giantswarm/cert-exporter)
```

No need for the token but we always use shield.

Reply:
Thanks, I was mislead by looking at an older kube-state-metrics-app commit, while there was followup one https://github.com/giantswarm/kube-state-metrics-app/commit/24969fd46fb061a17e959e41c507ded57f469eb4#diff-04c6e90faac2675aa89e2176d2eec7d8
Will fix same for net-exporter.